HISTORY.rst

```diff
@@ -5,6 +5,8 @@ History
 Unreleased
 ----------
 
+* **Breaking change:** glinter nits found during the `translate` command are now treated as errors, not warnings.
```

Comment:
Let's also call out that glinter `errors` are treated as errors, not `warnings` (even though we don't have them at the moment). Are you planning on having a follow-up bug to discuss what's an error and what's a warning?

Reply:
My thinking is that we would roll out new nits as warnings, and gradually move them to errors after users have had a chance to catch up.  It's possible on a case-by-case basis that some things would remain errors forever.

Reply:
That's fine! But in the long run, we're not going to break translate if we add new "warning classes", right?

Reply:
That's right.  Warnings won't cause translate to return an error code from the process.

Reply:
Cool, I'd just explicitly mention that in the changelog, for clarity for people outside of our group.